drivers/adc/ad738x/ad738x.h

```diff
@@ -177,6 +178,7 @@ struct ad738x_init_param {
 	/* Device Settings */
 	enum ad738x_conv_mode	conv_mode;
 	enum ad738x_ref_sel		ref_sel;
+	uint32_t		ref_voltage;
```

Comment:
some documentation should be given here regarding the measurement unit (volts or microvolts)

Reply:
how about i rename to  ref_voltage_mv?

Reply:
even better!